### PR TITLE
US6796: show pending requests to join

### PR DESCRIPTION
**Pull request by JoeKer1**

### CODE REVIEW

File: src/app/components/pin-details/gathering/gathering-requests/gathering-requests.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      spyOn(group, 'getGroupRequests').and.returnValue(Observable.of([{inquiryId: 1}, {inquiryId: 2}]));
+      this.component.ngOnInit();
+      expect(this.component.inquiries.length).toBe(2);
+      expect(group.getGroupRequests.calls.count()).toBe(1);
+    }));
+
+    it('should convert inquiry to participant', () => {
+        let inquiry = new Inquiry(1, 'theemail@email.com', null, 'Joe', 'Ker', new Date(2002), false, 42, 1, null);
+        let participant = this.component.convertToParticipant(inquiry);
+        console.log(participant);
```
Comment (vchtelmakh) [inline src/app/components/pin-details/gathering/gathering-requests/gathering-requests.component.spec.ts]: >test >console.log u wut m8?

File: src/app/components/pin-details/gathering/gathering-requests/gathering-requests.component.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    private groupService: GroupService,
+    private state: StateService
+  ) {
+  }
+
+  public ngOnInit() {
+      this.state.setLoading(true);
+    this.groupService.getGroupRequests(this.pin.gathering.groupId)
+        .subscribe(inquiryList => {this.inquiries = inquiryList;
+                                   this.state.setLoading(false)},
```
Comment (vchtelmakh) [inline src/app/components/pin-details/gathering/gathering-requests/gathering-requests.component.ts]: semicolon on 32

File: src/app/components/pin-details/gathering/gathering-requests/gathering-requests.component.spec.ts (hunk 1 above)
Comment (JoeKer1) [inline src/app/components/pin-details/gathering/gathering-requests/gathering-requests.component.spec.ts]: dang it

File: src/app/services/group.service.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  public defaults = {
+    authorized: null
+  };
+
+  constructor(private http: Http, private session: SessionService) {
+  }
+
+  public getGroupRequests(groupId: number): Observable<Inquiry[]> {
+    return this.session.get(`${this.baseUrl}api/v1.0.0/group-tool/inquiries/${groupId}`);
```
Comment (vchtelmakh) [inline src/app/services/group.service.ts]: y u no api service?!

**Pull request merged**